Tests/SonarScanner.MSBuild.PreProcessor.Test/AnalysisConfigGeneratorTests.cs

```diff
@@ -24,6 +24,7 @@
 using System.Linq;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
```

Comment:
Changes in this file:
* added `, null` argument before logger
* added one assertion highlighted by a message below
* everything else is cleanup